src/pages/workspaces/workspace/useDeleteWorkspaceState.ts

```diff
@@ -35,83 +53,153 @@ export interface DeleteWorkspaceHookArgs {
 }
 
 
-export const useDeleteWorkspaceState = (hookArgs: DeleteWorkspaceHookArgs) : DeleteWorkspaceState => {
+export const useDeleteWorkspaceState = (hookArgs: DeleteWorkspaceHookArgs): DeleteWorkspaceState => {
   const [deleting, setDeleting] = useState(false)
   const [loading, setLoading] = useState(false)
-  const [apps, setApps] = useState<DeleteWorkspaceModalLeoApp[]>()
   const [collaboratorEmails, setCollaboratorEmails] = useState<string[] | null>()
   const [workspaceBucketUsageInBytes, setWorkspaceBucketUsageInBytes] = useState<number | null>()
-  const [controlledResourcesExist, setControlledResourcesExist] = useState(false)
```

Comment:
I removed this state var for WSM controlled resources. Any non-deletable WSM resources (i.e., a leo controlled container or vm) should be covered by the leo resource deletion code -- any others are a bug at this point and should manifest as an error at deletion time.